plancheck/expect_known_output_value_at_path_test.go

```diff
@@ -140,6 +140,100 @@ func TestExpectKnownOutputValueAtPath_CheckPlan_AttributeValueNull(t *testing.T)
 	})
 }
 
+func TestExpectKnownOutputValueAtPath_CheckPlan_AttributeValueNotNull(t *testing.T) {
+	t.Parallel()
+
+	r.Test(t, r.TestCase{
+		ProviderFactories: map[string]func() (*schema.Provider, error){
+			"test": func() (*schema.Provider, error) { //nolint:unparam // required signature
+				return testProvider(), nil
+			},
+		},
+		// Prior to Terraform v1.3.0 a planned output is marked as fully unknown
+		// if any attribute is unknown. The id attribute within the test provider
```

Comment:
Since we know this information, it might be good to include it with the Go/website documentation surrounding output checks, if its not already. We'll want folks testing older versions of Terraform to know that outputs are fully unknown if any part of the value is unknown. 👍

Reply:
Have updated the Go docs and the website docs.